parse/compose.go

```diff
@@ -7,6 +7,22 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+func CatalogInfoFromTemplateVersion(contents []byte) (model.Version, error) {
+
+	var data map[string]interface{}
```

Comment:
Assuming the map in between here is for supporting camel case and underscore fields later on.